backend/python/app/graphql/types/story_type.py

```diff
@@ -49,6 +49,7 @@ class CreateStoryTranslationRequestDTO(graphene.InputObjectType):
     language = graphene.String(required=True)
     stage = graphene.Field(StageEnum, default_value="TRANSLATE")
 
+
```

Comment:
is this from the linter?

Reply:
yeah i think it adds two lines between each class declaration